#! /bin/sh /usr/share/dpatch/dpatch-run
## 014-default-password.dpatch by Joey Schulze <joey@infodrom.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix default password field name

diff -urNad mod-auth-mysql~/DIRECTIVES mod-auth-mysql/DIRECTIVES
--- mod-auth-mysql~/DIRECTIVES	2010-01-23 17:41:45.000000000 +0100
+++ mod-auth-mysql/DIRECTIVES	2010-01-23 17:47:02.000000000 +0100
@@ -124,7 +124,7 @@
 
 Auth_MySQL_Password_Field <password_field_name>
 	As per ...Username_Field above, but for passwords.  Same MySQL
-	access privileges.  Defaults to 'password'.
+	access privileges.  Defaults to 'passwd'.
 
 AuthMySQL_Password_Field <password_field_name>
 	Synonym for Auth_MySQL_Password_Field.
